src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -148,6 +148,12 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     }
   }
 
+  private updateMainFormWithChangesFromPageForm(editFormFields, pageFormFields) {
```

Comment:
As per conversation with Andrew this will only work for the top level (not in complex) field. This is what we do support at the moment and needs to be communicated with the ui-toolkit users.